lib/matplotlib/testing/__init__.py

```diff
@@ -50,7 +50,7 @@ def setup():
     set_reproducibility_for_testing()
 
 
-def subprocess_run_for_testing(command, env=None, timeout=None, stdout=None,
+def subprocess_run_for_testing(command, env=None, timeout=60, stdout=None,
                                stderr=None, check=False, text=True,
```

Comment:
```suggestion
                               stderr=None, check=True, text=True,
```
Since this is used for testing it seems like we should set `check=True` by default. It looks like you currently set this in many of the calls. Are there even more tests without setting check?